docs/migrating-v1beta1-to-v1.md

```diff
@@ -11,7 +11,8 @@ weight: 4000
 - [Upgrading `PipelineRun.Timeout` to `PipelineRun.Timeouts`](#upgrading-pipelinerun.timeout-to-pipelinerun.timeouts)
 - [Replacing Resources from Task, TaskRun, Pipeline and PipelineRun](#replacing-resources-from-task,-taskrun,-pipeline-and-pipelinerun)
 - [Replacing `taskRef.bundle` and `pipelineRef.bundle` with Bundle Resolver](#replacing-taskRef.bundle-and-pipelineRef.bundle-with-bundle-resolver)
-- [Adding `TaskRunTemplate` in `PipelineRun.Spec`](#adding-taskruntemplate-to-pipelinerun.spec)
+- [Replacing ClusterTask with Remote Resolution](#replacing-clustertask-with-remote-resolution)
```

Comment:
Wondering if we are expecting to add the section below?

Reply:
it's not needed and have added to navigate on that heading , should we remove it ?

Reply:
I think that looks fine as is now. Thanks!